src/Components/NavBar/MinimalNavBar.tsx

```diff
@@ -0,0 +1,33 @@
+import { ArtsyLogoBlackIcon, Box } from "@artsy/palette"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import React from "react"
+
+interface MinimalNavBarProps {
+  to: string
+  children: React.ReactNode
+}
+
+export const MinimalNavBar: React.FC<MinimalNavBarProps> = props => {
+  return (
+    <Box
+      zIndex={1000}
```

Comment:
Force's nav bar is like 980, which because of some other weird elements it has to be that high, so this is higher, because this just covers the page